doc/cephfs/index.rst

```diff
@@ -33,8 +33,19 @@ and Swift APIs, or native bindings (librados).
 Using CephFS
 ============
 
-Using the Ceph Filesystem requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Prerequisites for using CephFS with a running Ceph Storage Cluster:
+
+- At least one active :term:`Ceph Metadata Server`.
+- Minimum one data and metadata :term:`RADOS` pool.
```

Comment:
I don't think a list is necessary for this summary. It could be said:

> `Using CephFS requires configuring a number of Metadata Server (MDS) daemons <link to doc>, creating the file system <link to doc>, selecting a mount mechanism for clients <link to doc>, and configuring authentication credentials for clients <link to client-auth.rst>.`

Then you can also refer/link to the quick start document that @rishabh-d-dave is working on.

Reply:
Done.